src/pages/_app.tsx

```diff
@@ -35,6 +36,18 @@ const App = ({ Component, pageProps }: AppProps) => {
     }
   }
 
+  useEffect(() => {
+    OneSignal.init({
+      appId: process.env.NEXT_PUBLIC_ONESIGNAL_APP_ID!,
+      safari_web_id: process.env.NEXT_PUBLIC_ONESIGNAL_SAFARI_WEB_ID!,
```

Comment:
Environment key is missing in Doppler's dev environment. Unsure if it's the same as APP_ID or not